stable/ark/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "ark.fullname" . }}
+  release: {{ .Release.Name }}
```

Comment:
"Error: error validating \"\": error validating data: found invalid field release for v1.ObjectMeta"
I am getting the error above for `release` and for `app`.

Reply:
Missing labels. Should be something like:
```
metadata:
  name: {{ template "ark.fullname" . }}
  labels:
    app: {{ template "ark.name" . }}
    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
```

Reply:
Thanks, i'll add them now